cmd/realm/trigger_policy.go

```
@@ -0,0 +1,189 @@
+	Example: `  astartectl realm-management trigger-policy show my_trigger_policiy`,
+	Args:    cobra.ExactArgs(1),
+	RunE:    triggersPoliciesShowF,
+}
+
+var triggersPoliciesInstallCmd = &cobra.Command{
+	Use:   "install <trigger_file>",
+	Short: "Install trigger policy",
+	Long: `Install the given trigger policy in the realm.
+<trigger_file> must be a path to a JSON file containing a valid Astarte trigger policy.`,
+	Example: `  astartectl realm-management trigger-policy install my_policy.json`,
+	Args:    cobra.ExactArgs(1),
+	RunE:    triggersPoliciesInstallF,
+}
+
+var triggersPoliciesDeleteCmd = &cobra.Command{
+	Use:     "delete <trigger_name>",
+	Short:   "Delete a trigger policy",
+	Long:    `Deletes the specified trigger policy from the realm.`,
+	Example: `  astartectl realm-management trigger-policy delete my_trigger_policiy`,
+	Args:    cobra.ExactArgs(1),
+	RunE:    triggersPoliciesDeleteF,
+	Aliases: []string{"del"},
+}
+
+func init() {
+
+	RealmManagementCmd.AddCommand(triggersPoliciesCmd)
+	triggersPoliciesCmd.AddCommand(
+		triggersPoliciesListCmd,
+		triggersPoliciesShowCmd,
+		triggersPoliciesInstallCmd,
+		triggersPoliciesDeleteCmd,
+	)
+}
+
+func triggersPoliciesListtF(command *cobra.Command, args []string) error {
+	realmPolicies, _ := listPolicies(realm)
+	fmt.Println(realmPolicies)
+	return nil
+}
+
+func triggersPoliciesShowF(command *cobra.Command, args []string) error {
+	policyName := args[0]
+	policyDefinition, _ := getPolicyDefinition(realm, policyName)
+	respJSON, _ := json.MarshalIndent(policyDefinition, "", "  ")
+	fmt.Println(string(respJSON))
+	return nil
+}
+
+func triggersPoliciesInstallF(command *cobra.Command, args []string) error {
+	triggerFile, err := os.ReadFile(args[0])
+	if err != nil {
+		return err
+	}
+
+	var triggerPolicy map[string]interface{}
+	err = json.Unmarshal(triggerFile, &triggerPolicy)
+	if err != nil {
+		return err
+	}
+
+	_ = installTriggerPolicy(realm, triggerPolicy)
```

The `installTriggerPolicy` function can return an error, it should be checked instead of just printing `"ok"`